packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/Abstractions/ClientPipelineApi.cs

```diff
@@ -26,7 +28,8 @@ protected ClientPipelineApi(Type type, ValueExpression original) : base(type, or
 
         public abstract ValueExpression Create(ValueExpression options, ValueExpression perRetryPolicies);
 
-        public abstract ValueExpression AuthorizationPolicy(params ValueExpression[] arguments);
```

Comment:
What is the reasoning behind changing from naming this to be a policy. SCM and Azure.Core have the concept of policies and I think most if not all implementers would use a policy. If they didn't want to use the policy primitive, they can still return whatever ValueExpression they want, but it is better IMO if we guide implementers towards the best practice.

Reply:
In my initial version, this is `KeyAuthorizationPolicy` and `TokenAuthorizationPolicy`.
Which way do you like?
We could call it like:
`CreateKeyAuthorizationPolicy` and `CreateOAuth2AuthorizationPolicy`?

Reply:
I think the simple KeyAuthorizationPolicy and TokenAuthorizationPolicy works. We can go through and polish all the names once we are closer to releasing a beta.

Reply:
renamed to KeyAuthorizationPolicy and TokenAuthorizationPolicy